tests/main/refresh/task.yaml

```diff
@@ -44,3 +44,14 @@ execute: |
   echo "Then the new version is available for the snap to be refreshed"
   expected="$SNAP_NAME +\d+\.\d+\+fake1 +\d+ +canonical"
   echo "$(snap refresh --list)" | grep -Pzq "$expected"
+
+  echo "================================="
+
+  echo "When the snap is refreshed"
+  snap refresh $SNAP_NAME
+
+  echo "Then the new version is listed"
+  expected="$SNAP_NAME +\d+\.\d+\+fake1 +\d+ +canonical"
+  echo "$(snap list)" | grep -Pzq "$expected"
```

Comment:
Any reason for this to be `echo $(foo) | ...` instead of `foo | ...`?

Let's please fix all occurrences on all tests.